### PR TITLE
Moved _issue_command to BrowserManagerHandle

### PR DESCRIPTION
**Pull request by vringar**
This piece of code conceptually belongs to the BrowserManagerHandle, so I moved it there.
However I couldn't think of an easy way to remove all references to the TaskManager so for now we still pass the TaskManager as a parameter.

### CODE REVIEW

File: openwpm/browser_manager.py [hunk 1]
```diff
@@ -11,19 +12,28 @@
...
 from pathlib import Path
 from queue import Empty as EmptyQueue
-from typing import Optional, Union
+from typing import TYPE_CHECKING, Optional, Tuple, Union
 
 import psutil
+import tblib
 from multiprocess import Queue
 from selenium.common.exceptions import WebDriverException
 from tblib import pickling_support
```
Comment (boolean5) [inline openwpm/browser_manager.py:21]:
```suggestion
```

File: openwpm/browser_manager.py [hunk 2]
```diff
@@ -34,9 +44,9 @@
 pickling_support.install()
```
Comment (boolean5) [inline openwpm/browser_manager.py:44]:
```suggestion
tblib.pickling_support.install()
```

**Review by boolean5 — CHANGES_REQUESTED** [review]

**Comment by boolean5** [comment]
We should also update the docs wherever there are references to the `Browser` class. With a quick look I just found [one](https://github.com/mozilla/OpenWPM/blame/master/docs/Platform-Architecture.md#L62) in `Platform-Architecture.md`.

**Comment by vringar** [comment]
Thank you so much for looking into this and figuring out where my error was.

**Review by boolean5 — APPROVED** [review]

**Pull request merged**